elastic/kibana
# [Response Ops] Allow `_source` field for ES DSL query rules

Resolves https://github.com/elastic/kibana/issues/124749

## Summary

Allows users to specify `_source` field in their ES DSL query in order to filter the fields that are returned.

## To Verify

Create an ES query rule type using the different `_source` options available: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-fields.html#source-filtering. Ensure that the source returned in the `context.hits` variable is as expected. I used a server log action to log the `context.hits` array

### Checklist
